core/server/master/src/main/java/alluxio/master/file/meta/InodeDirectory.java

```diff
@@ -308,7 +317,7 @@ public JournalEntry toJournalEntry() {
         .setTtl(getTtl())
         .setTtlAction(ProtobufUtils.toProtobuf(getTtlAction()))
         .setDirectChildrenLoaded(isDirectChildrenLoaded())
-        .setOwner(getOwner()).setGroup(getGroup()).setMode(getMode())
+        .setAcl(AccessControlList.toProtoBuf(mAcl))
```

Comment:
This means an older master cannot read a newer journal?

Reply:
yes, but newer master can read old journal.